NebulaNetwork/Ngrok/NgrokManager.cs

```diff
@@ -298,9 +298,8 @@ public bool IsNgrokActive()
             _ngrokProcess?.Refresh();
             return !_ngrokProcess?.HasExited ?? false;
         }
-        catch (Exception e)
+        catch
         {
-            Log.Error(e);
```

Comment:
Do we not want to log cases where there is an error?

Reply:
Does logging of [InvalidOperationException](https://learn.microsoft.com/en-us/dotnet/api/system.invalidoperationexception?view=netframework-4.7.2) of `Process.HasExited` really help debugging?
Wouldn't [NgrokLastErrorCode](https://github.com/NebulaModTeam/nebula/blob/master/NebulaNetwork/Ngrok/NgrokManager.cs#L120-L124) more helpful?

Reply:
Well the ngroklasterrorcode will only get populated if the error originates from the ngrok process directly. This check was because of ppl getting errors/crashes/etc because the process wasn't started properly at all (e.g. anti-virus quarantined the ngrok exe)